pages/dao/[symbol]/proposal/new.tsx

```diff
@@ -547,6 +557,16 @@ const New = () => {
       [Instructions.RemoveKeyFromDID]: RemoveKeyFromDID,
       [Instructions.AddServiceToDID]: AddServiceToDID,
       [Instructions.RemoveServiceFromDID]: RemoveServiceFromDID,
+      [Instructions.UXDInitializeController]: UXDInitializeController,
```

Comment:
please sort by alphabetical order

Reply:
Isn't it better to have them grouped by thematic? Controller interactions/Depository interactions/ Mint-Redeem